myapp/Gemfile

```diff
@@ -49,6 +51,7 @@ group :development do
   gem 'listen', '~> 3.2'
   gem 'web-console', '>= 3.3.0'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'bullet'
```

Comment:
Since you add bullet to development and test env, please implement bullet to rspec.